### PR TITLE
[KIECLOUD-547] - Prepare RHPAM Kogito Operator for 7.11.1 release

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: spolti <fspolti@redhat.com>
See: https://issues.redhat.com/browse/KIECLOUD-547
Many thanks for submitting your Pull Request :heart:! 

Please make sure your PR meets the following requirements:

- [x] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/master/README.md#contributing-to-the-kogito-operator)
- [x] Pull Request contains a link to the JIRA issue
- [x] Pull Request contains a description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [x] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](RELEASE_NOTES.md) entry regarding this change

### CODE REVIEW

File: rhpam-kogito-operator.yaml [hunk 1]
```diff
@@ -49,81 +49,54 @@ spec:
     name: v1
     schema:
       openAPIV3Schema:
-        description: KogitoBuild handles how to build a custom Kogito service in a
-          Kubernetes/OpenShift cluster.
+        description: KogitoBuild handles how to build a custom Kogito service in a Kubernetes/OpenShift cluster.
```
Comment (ricardozanini) [inline rhpam-kogito-operator.yaml:52]: I think we might consider deleting this file. Or would users install it manually from this git project?

**Review by ricardozanini — APPROVED** [review]

**Review by radtriste — APPROVED** [review]
lgtm

File: rhpam-kogito-operator.yaml (hunk 1 above)
Comment (spolti) [reply rhpam-kogito-operator.yaml:52]: not sure, probably better keep it around for a while.

**Review by AntStephenson — APPROVED** [review]
LGTM

**Pull request merged**